app/src/main/java/com/mk/tasky/agenda/data/mapper/EventMapper.kt

```diff
@@ -0,0 +1,36 @@
+package com.mk.tasky.agenda.data.mapper
+
+import com.mk.tasky.agenda.data.local.entity.EventEntity
+import com.mk.tasky.agenda.domain.model.AgendaItem
+import com.mk.tasky.agenda.util.TimeUtil.longToTime
+import com.mk.tasky.agenda.util.TimeUtil.timeToLong
+
+fun AgendaItem.Event.toEntity(): EventEntity {
+    return EventEntity(
+        eventId = this.eventId,
+        title = this.eventTitle,
+        description = this.eventDescription,
+        remindAt = timeToLong(this.eventRemindAt),
```

Comment:
In this case, I'd find an extension function a bit more readable: `this.eventRemindAt.toLong()`
If someone doesn't know this `timeToLong` function exists in the project, they might re-implement it here. But an extension function is likely to be found in the suggestions when writing `dateTime.`